airflow/serialization/serialized_objects.py

```diff
@@ -553,6 +553,7 @@ def serialize(
         elif isinstance(var, MappedOperator):
             return cls._encode(SerializedBaseOperator.serialize_mapped_operator(var), type_=DAT.OP)
         elif isinstance(var, BaseOperator):
+            var.needs_expansion = var.get_needs_expansion()
```

Comment:
```suggestion
            var._needs_expansion = var.get_needs_expansion()
```

I think? (Why do the static checks not complain… I feel like I’m missing something.)

Reply:
it's a private attr, that we manually set at serialization time.

Reply:
No but setting `var.needs_expansion` would not work right?

Reply:
oh dang yeah sorry, that was a pre-coffee response